lib/plugins/creative.js

```diff
@@ -2,12 +2,14 @@ const assert = require('assert')
 const Vec3 = require('vec3').Vec3
 const { callbackify, sleep } = require('../promise_utils')
 const { once } = require('events')
+const { Physics } = require('prismarine-physics')
 
 module.exports = inject
 
 function inject (bot, { version }) {
   const Item = require('prismarine-item')(version)
-
+  const mcData = require('minecraft-data')(version)
+  const constantGravity = Physics(mcData).gravity
```

Comment:
Can't we just do that:

```js
  const constantGravity = bot.physics.gravity
```

Or set it in physics plugin? Like `bot._constantGravity` and then use it in `stopFlying`?

Reply:
Ok I'll just do that then

Reply:
It was a comment for a discussion, not a request to do changes :)

Simply setting `constantGravity` to `bot.physics.gravity` is actually kind of dangerous, as physics plugin may not load before creative. In such case, it'll throw an exception that bot.physics is undefined. The second approach is a bit better but still should be discussed.